samples/bluetooth/direction_finding_peripheral/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _direction_finding_peripheral:
+
+Bluetooth: Direction finding peripheral
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The direction finding peripheral sample demonstrates Bluetooth® LE direction finding transmission as a response to request received from connected central device.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns
+
+The sample also requires an antenna matrix when operating in angle of departure mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix, or any other antenna matrix.
+
+Overview
+********
+
+The direction finding connectionless beacon sample application uses Constant Tone Extension (CTE), that is transmitted with periodic advertising PDUs.
```

Comment:
Could we use sample instead of sample application for all?

Reply:
Yes, I've removed the `application` word from the sentence.
BTW. The sentence should have been: `The direction finding peripheral sample uses Constant Tone Extension (CTE), that is transmitted with periodic advertising PDUs.`